tests/test_routes.py

```diff
@@ -85,6 +89,7 @@ def test_create_application_creates_unique_reference(
         "account_id": "usera",
         "fund_id": "47aef2f5-3fcb-4d45-acb5-f0152b5f03c4",
         "round_id": "c603d114-5364-4474-a0c4-c41cbf4d3bbd",
+        "language": "en",
```

Comment:
See previously comment, this won't happen on production, so the test data should include some `null` values and the application should still work and default in that case to `en`.

Reply:
have update test data to reflect all cases of en, null and cy - will add logic to pull english forms if language = null and add more tests to cover this scenario 👍🏾